xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -517,10 +517,9 @@ bool CAESinkAUDIOTRACK::Initialize(AEAudioFormat &format, std::string &device)
 
       // TrueHD needs a smaller buffer (in duration) to reduce latency
       // this fixes dropouts since the data arrives in smaller quantities but more constantly
-      const double target_duration =
-          (m_passthrough && m_format.m_streamInfo.m_type == CAEStreamInfo::STREAM_TYPE_TRUEHD)
-              ? 0.08
-              : 0.15;
+      double max_time = 0.064;
+      double target_duration = 0.128;
```

Comment:
As commented in forum and Slack this change "128 ms for all" is fine and also I already tested in my side for a week so no risk of regressions here.

Only better change to `constexpr double`  and remove two lines of comment:

` // TrueHD needs a smaller buffer (in duration) to reduce latency....`

Since this workaround is no longer necessary as the root cause has been solved with https://github.com/xbmc/xbmc/pull/22392

Reply:
Will do - sure.